### DISCUSSION TITLE
How do i change the decimal notation to be a dot instead of a comma?

### DISCUSSION THREAD

**Discussion by ErikoStrand** [barvian/number-flow#69] [Help] [answered]
![image](https://github.com/user-attachments/assets/20a64339-4312-4615-93ac-eb3b4f4e2317)

**Comment by barvian** [comment]
Hi 👋, could you post your `locales` and `format` props (if any)?

**Reply by ErikoStrand** [reply]
thats the whole numbeflow. is it compact thats messing with it?
`<NumberFlow
    class="h-8"
    format={{ notation: 'compact' }}
    value={generalData.averageMediaPerMonth}
/>`

**Reply by barvian** [reply]
Hi, what if you use a different locale? e.g. `<NumberFlow class="h-8" locales="en-US" format={{ notation: 'compact' }} value={generalData.averageMediaPerMonth} />`

**Reply by ErikoStrand** [reply]
sorry for the late response, Doesnt change anything sadly :(

`format={{ notation: 'compact' }}
						locales="en-US"
						value={generalData.averageMediaPerMonth}`

**Reply by barvian** [reply]
Hi, can you reproduce it in [this sandbox](https://codesandbox.io/p/sandbox/number-flow-69-lqwc4d)? I'm not seeing the comma there.

**Comment by ErikoStrand** [comment, marked as answer]
Oh no sorry im a bit behind in the brain department, It would seem like you were correct from the start and the locales="en-US" worked I just missplaced the statement in the other one where there isnt a comma and thought it was on the right one, noticed my mistake sorry for taking your time. And thanks for the quick help <3

(for anyone else Locales="en-US" is correct).
